### ISSUE TITLE
`helm-exwm-buffer-max-length` cannot be set to `nil`

### ISSUE THREAD

**Issue by aciceri** [emacs-helm/helm-exwm#9]
If I try to set `helm-exwm-buffer-max-length` to `nil` then when I use `helm-exwm` I get the following error:

```
Wrong type argument: number-or-marker-p, nil
```

In the description of the variable is written that

> Max length of EXWM buffer names before truncating.
> When disabled (nil) use the longest buffer-name length found.
> See `helm-buffer-max-length`.  This variable's default is so that
> the EXWM class starts at the column of the open parenthesis in
> `helm-buffers-list' detailed view.

so I assume I can do what I'm trying to do. I just started using exwm and helm and I'm a beginner with Emacs but I can try to study the script if this is actually a bug.
I'm using Emacs 26.3 with the last versions of the packages from MELPA.

Thanks

**Comment by thierryvolpiatto** [comment]
I don't use helm-exwm nor exwm, but the code seems to come from helm-buffers.el.
In helm-buffers.el, I initialize the longest buffer-name by looping through all buffers and then, if `helm-buffer-max-length` is unset (nil) I give it a default value (locally) equal to the longest buffer-name calculated before, like this:

```elisp
(unless (default-value 'helm-buffer-max-length)
      (helm-set-local-variable 'helm-buffer-max-length (car result)))
```
@Ambrevar, seems you forget to do this in helm-exwm (or I missed something?) hence the error @andrea96  have.

**Comment by Ambrevar** [comment]
Fixed in 111ac4c37f19ec8b65a0bbae5f6b06a4b5f1a05b.  Thanks for reporting!
